### PR TITLE
Miscellaneous tweaks made while getting up to speed on the repo

### PR DESCRIPTION
**Pull request by mhuesch**
Small changes to a few files.

### CODE REVIEW

File: src/Cardano/Chain/Genesis/Config.hs [hunk 1]
```diff
@@ -113,14 +114,13 @@ instance FromJSON StaticConfig where
...
-            kV
-            protocolMagicV
-            initializerV
+        either fail (pure . GCSpec) $ mkGenesisSpec
+          avvmDistrV
+          heavyDelegationV
+          protocolParametersV
+          kV
+          protocolMagicV
+          initializerV
```
Comment (mhuesch) [inline src/Cardano/Chain/Genesis/Config.hs:123]: I made this edit on the assumption that we shouldn't ever have invalid datatypes specified in config. It's that we want that, but it seems weird. If that is indeed the case, we should document why here.

Comment (intricate) [reply src/Cardano/Chain/Genesis/Config.hs:123]: Off-the-top, I can't imagine why we'd want to construct an invalid `GCSpec` so this seems like a good modification to me.

File: src/Cardano/Chain/Genesis/AvvmBalances.hs [hunk 2]
```diff
@@ -19,7 +19,10 @@ import Cardano.Chain.Common (Lovelace)
 import Cardano.Crypto.Signing.Redeem (RedeemPublicKey)
 
 
--- | Predefined balances of avvm entries.
+-- | Predefined balances of AVVM (Ada Voucher Vending Machine) entries.
```
Comment (intricate) [inline src/Cardano/Chain/Genesis/AvvmBalances.hs:22]: Good addition. Will definitely save someone the time of having to look into this.

**Review by intricate — APPROVED** [review]

File: src/Cardano/Chain/Common/LovelacePortion.hs [hunk 3]
```diff
@@ -70,6 +70,9 @@ instance Bi LovelacePortion where
   encode = encode . getLovelacePortion
   decode = LovelacePortion <$> decode
 
+-- The Canonical and Aeson instances for LovelacePortion are inconsistent -
```
Comment (dnadales) [inline src/Cardano/Chain/Common/LovelacePortion.hs:73]: Nice :)

**Review by dnadales — APPROVED** [review]
Thanks for cleaning things a bit, and adding the comments!

File: src/Cardano/Chain/Common/LovelacePortion.hs (hunk 3 above)
Comment (ruhatch) [reply src/Cardano/Chain/Common/LovelacePortion.hs:73]: :+1:

File: src/Cardano/Chain/Genesis/AvvmBalances.hs [hunk 4]
```diff
@@ -19,7 +19,10 @@ import Cardano.Chain.Common (Lovelace)
...
+-- People who purchased Ada at a pre-sale were issued a certificate at the
```
Comment (ruhatch) [inline src/Cardano/Chain/Genesis/AvvmBalances.hs]:
```suggestion
-- People who purchased Ada at a pre-sale were issued a certificate
```

File: src/Cardano/Chain/Genesis/Config.hs (hunk 1 above)
Comment (ruhatch) [reply src/Cardano/Chain/Genesis/Config.hs:123]: Seems good :)

**Review by ruhatch — APPROVED** [review]
Thanks @mhuesch

**Review by Jimbo4350 — APPROVED** [review]
LGTM!

File: src/Cardano/Chain/Genesis/AvvmBalances.hs (hunk 4 above)
Comment (mhuesch) [reply src/Cardano/Chain/Genesis/AvvmBalances.hs]: Thanks for the grammar catch. I just dropped the extra words and then rewrapped the text: https://github.com/input-output-hk/cardano-ledger/pull/389/files#diff-633568df21fc9c3cf311dbc4c323a689R24

**Pull request merged**